### PR TITLE
Update some Servers

### PR DESCRIPTION
**Pull request by l3nnartt**
## Type of change

- [✅] Update directory of a server

## Checklist

- Update AntiAC
- Update Buildmania
- Update Cytooxien
- Update MyPvP

### CODE REVIEW

File: minecraft_servers/buildmania/manifest.json [hunk 1]
```diff
@@ -7,8 +7,13 @@
...
     "social": {
       "web": "https://buildmania.cz",
+      "web_shop": "https://store.buildmania.cz/",
       "discord": "https://discord.buildmania.cz",
       "instagram": "buildmanianetwork",
       "tiktok": "buildmanianetwork"
+    },
+    "discord": {
+      "server_id": 651073986716893194,
+      "rename_to_minecraft_name": true
```
Comment (JakobBruening) [inline minecraft_servers/buildmania/manifest.json:17]: Please remove the line here too!

File: minecraft_servers/cytooxien/manifest.json [hunk 2]
```diff
@@ -8,7 +8,15 @@
...
+    "web_support": "https://www.cytooxien.de/support/",
     "twitter": "Cytooxien",
-    "discord": "https://cxn.link/discord"
-  }
+    "discord": "https://cxn.link/discord",
+    "teamspeak": "ts.cytooxien.net"
+  },
+  "discord": {
+    "server_id": 443768116216987648,
+    "rename_to_minecraft_name": true
```
Comment (JakobBruening) [inline minecraft_servers/cytooxien/manifest.json:19]: Please remove the line here too!

File: minecraft_servers/mypvp/manifest.json [hunk 3]
```diff
@@ -17,8 +17,15 @@
...
+    "web_support": "https://mypvp.me/forum/index.php?board/9-support/",
     "twitter": "MyPvPme",
     "youtube": "https://www.youtube.com/channel/UCq2tbnNIT9ANK91uFExFC3g",
-    "discord": "https://discord.com/invite/uPXmSs6"
+    "discord": "https://discord.com/invite/uPXmSs6",
+    "teamspeak": "mypvp.me"
+  },
+  "discord": {
+    "server_id": 633080118897803284,
+    "rename_to_minecraft_name": true
```
Comment (JakobBruening) [inline minecraft_servers/mypvp/manifest.json:29]: Please remove the line here too!

**Review by JakobBruening — CHANGES_REQUESTED** [review]

File: minecraft_servers/cytooxien/manifest.json (hunk 2 above)
Comment (l3nnartt) [reply minecraft_servers/cytooxien/manifest.json:19]: I guess Cytooxien ist a partner Server 🤔 

https://labymod.net/en/partner

https://i.lennartloesche.de/ROJo3/duBoWaLa91.png

**Review by l3nnartt — COMMENTED** [review]
Remove Discord (Partner Feature)

File: minecraft_servers/cytooxien/manifest.json (hunk 2 above)
Comment (JakobBruening) [reply minecraft_servers/cytooxien/manifest.json:19]: Yes. You are very welcome to fill out the `server_id` in (it's good if we have the data). But please take out "`rename_to_minecraft_name`". This is for team members of the server to decide :)

**Review by JakobBruening — CHANGES_REQUESTED** [review]

**Review by JakobBruening — APPROVED** [review]
Thank you very much for your contribution! 🌟

**Pull request merged**